packages/babel-parser/test/fixtures/typescript/const/no-initializer/output.json

```diff
@@ -1,6 +1,9 @@
 {
```

Comment:
Can you create a test with the same input as this one in the `dts` folder?

Reply:
Done: https://github.com/tolmasky/babel/commit/d8c403a9c29b53549c73d7360c05bf934b6038dc